docs/developer-guide.md

```diff
@@ -89,6 +89,8 @@ You can now build RiseDev and start a dev cluster. It is as simple as:
 psql -h localhost -p 4566 -d dev -U root
 ```
 
+If you face memory bottlenecks while compiling, either partition some space on your computer as SWAP or lower compilation parallelism with [`CARGO_BUILD_JOBS`](https://doc.rust-lang.org/cargo/reference/config.html#buildjobs), e.g. `CARGO_BUILD_JOBS=2`.
```

Comment:
Looks good! There's one slight wording adjustment that I would suggest:
--> If you detect memory bottlenecks while compiling, either allocate some disk space on your computer as swap memory, or lower the compilation parallelism with ...